File: javascript/mutations/cancelScheduleRequest.js
const { gql } = require("graphql-request")
const client = require("../utils/graphqlClient")

const query = gql`
mutation cancelScheduleRequest($inputs: CancelScheduleRequestInputs!) {
  cancelScheduleRequest(inputs: $inputs) {
    errorCode
  }
}
`

client.request(
  query,
  {
    inputs: {
      scheduleRequestId: '<Insert Your Schedule Request ID>'
    }
  }
).then((data) => console.log(data))

File: ruby/mutations/cancel_schedule_request.rb
require_relative "../utils/graphql_client"

CancelScheduleRequestMutation = Client.parse <<-'GRAPHQL'
  mutation ($inputs: CancelScheduleRequestInputs!) {
    cancelScheduleRequest(inputs: $inputs) {
      errorCode
    }
  }
GRAPHQL

result = Client.query(
  CancelScheduleRequestMutation,
  variables: {
    inputs: {
      scheduleRequestId: '<Insert Your Schedule Request ID>'
    }
  }
)
pp result.data.to_h

